libp2p/wire.nim

```diff
@@ -213,3 +213,13 @@ proc getLocalAddress*(sock: AsyncFD): TransportAddress =
   if getsockname(SocketHandle(sock), cast[ptr SockAddr](addr saddr),
                  addr slen) == 0:
     fromSAddr(addr saddr, slen, result)
+
+proc isPublicMA*(ma: MultiAddress): bool =
+  if DNS.matchPartial(ma):
+    return true
+
+  let hostIP = initTAddress(ma)
+  if hostIP.isOk() and hostIP.get().isGlobal():
+    return true
+
+  return false
```

Comment:
```suggestion
  let hostIP = initTAddress(ma).valueOr: return false
  return hostIp.isGlobal()
```

Reply:
There's a typo in `hostIp`, but pushed a commit with it.